Lib/traceback.py

```diff
@@ -1095,6 +1095,30 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
                         self._str += f" Or did you forget to import '{wrong_name}'?"
                     else:
                         self._str += f". Did you forget to import '{wrong_name}'?"
+        elif exc_value and isinstance(exc_value, UnpackError):
```

Comment:
Should it really be here? It appears that we will have duplicated messages and I'm not sure it should be done as such. What I mean is that I don't think it's the TracebackException responsibility to reformat this thing. We should treat `UnpackError` as any other normal error. If an exception in `__len__` happens, then let it propagate normally.

Reply:
Resolved.